src/Platform.Handlers/src/Xamarin.Platform.Handlers/Core/ILabel.cs

```diff
@@ -1,10 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using Xamarin.Forms;
 
 namespace Xamarin.Platform
 {
-	public interface ILabel : IText
+	public interface ILabel : IText, IPadding
```

Comment:
Can we remove all the properties we're not implementing?

Reply:
Sure. Added all the properties to include all the methods in the Mapper and easily know what is pending, do we delete or leave these properties and empty methods?

Reply:
Removed!